File: android/EpubRnD/src/com/hurix/epubRnD/Constants/GlobalConstants.java
package com.hurix.epubRnD.Constants;

import java.io.File;

import com.hurix.epubRnD.Utils.Constants;

import android.os.Environment;

public class GlobalConstants 
{
	public static final String ROOT_FOLDER = Environment.getExternalStorageDirectory()+File.separator+Constants.EPUBBOOK;

	public static int MIN_FONT_SIZE = 24;
	public static int MAX_FONT_SIZE = 36;
	public static int FONT_SIZE_STEP_SIZE = 2;

	public static int PAGE_COUNT = 0;
	//ePub type
	public static final int REFLOWABLE = 100;
	public static final int FIXED = 200;

	public static boolean ENABLE_WEB_VIEW_TOUCH = false;
	
	
	public static int NOTE_ICON_WIDTH = 40; //assuming base dpi as 160
	public static int NOTE_ICON_HEIGHT = 40; //assuming base dpi as 160
}

File: ios/EpubRnD/EpubRnD/GlobalConstants.m
//
//  Constants.m
//  EpubRnD
//
//  Created by UdaySravan K on 11/06/14.
//  Copyright (c) 2014 hurix. All rights reserved.
//

#import "GlobalConstants.h"



@implementation GlobalConstants

NSInteger const MAX_FONT_SIZE = 178;
NSInteger const MIN_FONT_SIZE = 150;
NSInteger const FONT_SIZE_STEP_SIZE = 4;

//EPUB type
NSInteger const REFLOWABLE = 100;
NSInteger const FIXED = 200;

NSInteger const DEFAULT_STICKYNOTE_WIDTH = 40;
NSInteger const DEFAULT_STICKYNOTE_HEIGHT = 40;

NSInteger const DEFAULT_BOOKMARK_WIDTH = 30;
NSInteger const DEFAULT_BOOKMARK_HEIGHT = 60;

NSInteger const PAGE_INDEX_GREATER_THAN_PAGE_COUNT = -2;

NSInteger const GET_PAGE_INDEX_USING_WORD_ID = -3;

NSString *const CHAPTERS_FOLDER_PATH = @"assets/cole-voyage-of-life-20120320/EPUB/xhtml";
//NSString *const CHAPTERS_FOLDER_PATH = @"assets/3920131231111/OPS";
@end
